lib/pbench/test/unit/server/test_shell_cli.py

```diff
@@ -251,10 +262,15 @@ def run(args, cwd: Optional[str] = None) -> subprocess.CompletedProcess:
 
     @staticmethod
     def test_main_crontab_failed(monkeypatch, make_logger, mock_get_server_config):
+        def noop(*args, **kwargs):
+            pass
+
         def generate_crontab_if_necessary(*args, **kwargs) -> int:
             return 43
 
         monkeypatch.setattr(shell.site, "ENABLE_USER_SITE", False)
+        monkeypatch.setattr(shell, "wait_for_uri", noop)
+        monkeypatch.setattr(shell, "init_indexing", noop)
```

Comment:
`noop` describes what the _mock does_, but it doesn't really convey what its effect on the CUT is supposed to be.  I suggest "immediate_succes" or something similar.

Ditto line 287.

Reply:
Fixed.